src/main/scala/org/apache/spark/ml/commons/GaussianProcessCommons.scala

```diff
@@ -6,16 +6,16 @@ import org.apache.spark.ml.commons.kernel.{EyeKernel, Kernel, _}
 import org.apache.spark.ml.commons.util.DiffFunctionMemoized
 import org.apache.spark.ml.feature.LabeledPoint
 import org.apache.spark.ml.linalg.Vector
-import org.apache.spark.ml.util.Instrumentation
 import org.apache.spark.ml.{PredictionModel, Predictor}
 import org.apache.spark.rdd.RDD
 import org.apache.spark.sql.functions.col
 import org.apache.spark.sql.{Dataset, Row}
 
 private[ml] trait GaussianProcessCommons[F, E <: Predictor[F, E, M], M <: PredictionModel[F, M]]
-  extends ProjectedGaussianProcessHelper {  this: Predictor[F, E, M] with GaussianProcessParams =>
+  extends ProjectedGaussianProcessHelper {
+  this: Predictor[F, E, M] with GaussianProcessParams =>
```

Comment:
What's the point of this change? Was the original line longer than 120 characters?